Network/Socket/Win32/Cmsg.hsc

```diff
@@ -149,24 +149,24 @@ instance ControlMessage IPv6TClass where
 ----------------------------------------------------------------
 
 -- | Network interface ID and local IPv4 address.
-data IPv4PktInfo = IPv4PktInfo ULONG HostAddress deriving (Eq)
+data IPv4PktInfo = IPv4PktInfo Int HostAddress HostAddress deriving (Eq)
```

Comment:
Perhaps a comment here indicating that we're diverging from the struct on purpose to align the interface with linux.

Reply:
Done.